src/components/footer/footer.js

```diff
@@ -34,68 +49,81 @@ export default function footer({ data }) {
         }
       `}
       render={data => (
-        <FooterContainer>
-          <FooterMainContent>
-            <SocialSection>
-              <FooterHeader>Powerhouse</FooterHeader>
-              <AddressSection>
-                123 Lorem Ipsum Dolor Tempor Incididunt YZ, 12345 123-456-7890
-              </AddressSection>
-            </SocialSection>
+        <FooterSection>
+          <FooterImage>
+            <img alt="footer background image" src={data.prismicFooter.data.footer_image.url} />
```

Comment:
Can you be a little more descriptive in the alt text please?
https://moz.com/learn/seo/alt-text